tests/manager/api/controller/test_patron_access_token.py

```diff
@@ -1,66 +1,72 @@
-from typing import TYPE_CHECKING
+from unittest.mock import MagicMock
 
 import pytest
 from werkzeug.datastructures import Authorization
 
+from palace.manager.api.controller.patron_auth_token import PatronAuthTokenController
 from palace.manager.api.problem_details import PATRON_AUTH_ACCESS_TOKEN_NOT_POSSIBLE
-from tests.fixtures.api_controller import CirculationControllerFixture
 from tests.fixtures.database import DatabaseTransactionFixture
-from tests.fixtures.services import ServicesFixture
+from tests.fixtures.flask import FlaskAppFixture
 
-if TYPE_CHECKING:
-    from palace.manager.api.controller.patron_auth_token import (
-        PatronAuthTokenController,
-    )
 
-
-class PatronAuthTokenControllerFixture(CirculationControllerFixture):
-    def __init__(
-        self, db: DatabaseTransactionFixture, services_fixture: ServicesFixture
-    ) -> None:
-        super().__init__(db, services_fixture)
-        self.controller: PatronAuthTokenController = self.manager.patron_auth_token
+class PatronAuthTokenControllerFixture:
+    def __init__(self, db: DatabaseTransactionFixture) -> None:
+        mock_manager = MagicMock()
+        mock_manager._db = db.session
+        self.controller = PatronAuthTokenController(mock_manager)
 
 
 @pytest.fixture(scope="function")
-def patron_auth_token_fixture(
-    db: DatabaseTransactionFixture, services_fixture: ServicesFixture
-):
-    return PatronAuthTokenControllerFixture(db, services_fixture)
+def patron_auth_token_fixture(db: DatabaseTransactionFixture):
+    return PatronAuthTokenControllerFixture(db)
 
 
 class TestPatronAuthTokenController:
```

Comment:
All the test changes here are to update these tests to use `FlaskAppFixture`, so we can set the patron by passing it to `flask_app_fixture.test_request_context`. The tests should be functionally the same.